docs/package.json

```diff
@@ -3,6 +3,6 @@
     "mintlify": "^4.0.122"
   },
   "dependencies": {
-    "sharp": "^0.33.2"
+    "sharp": "^0.33.4"
```

Comment:
why? lol can you revert? this feels werid

Reply:
sure